app/services/c100_app/screener_decision_tree.rb

```diff
@@ -18,12 +20,24 @@ def check_if_court_is_valid
       if courts.empty?
         show(:no_court_found)
       else
-        edit('/steps/miam/consent_order')
+        # store the first court in session so that if they choose "yes, it's urgent"
+        # we can render the court's contact details out to them in the next step
+        court = Court.new.from_courtfinder_data!(courts.first)
+        c100_application.screener_answers.update!(local_court: court)
```

Comment:
I have the feeling this might belong better to the form object (the one for Postcode). So the form might validate the postcode, using the API, and also store the address. But it is fine for now, there is so much to do 😭

Reply:
hmm, there's a discussion to be had there - IMHO the logic for hitting the API, handling the results, and taking appropriate action based on them, should not be in the form object, it's more of a controller-level concern. 

Let's come back to that :)